src/test/java/racingcar/StepOne.java

```diff
@@ -0,0 +1,68 @@
+package racingcar;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.*;
+
+class StepOne {
+
+    @Test
+    void StringTest() {
```

Comment:
관례상 메소드 이름은 소문자로 시작합니다.
```suggestion
    void stringTest() {
```